client/src/components/RestaurantInfo.vue

```diff
@@ -0,0 +1,50 @@
+<script setup lang="ts">
+import { onMounted, ref } from 'vue'
+import apis, { type Eatery } from '../lib/apis'
+import useParam from '../lib/param'
+
+const eateryId = useParam('eateryId')
+
+interface Restaurant {
+  name: string
+  description: string
+  id: number
+}
+
+const shopDetail = ref<Restaurant>({
+  name: 'ぷぐま',
+  description: '普通です',
+  id: 0,
+})
+
+const eateryDetail = ref<Eatery>()
+onMounted(async () => {
+  eateryDetail.value = (await apis.eateriesEateryIdGet(eateryId.value)).data
+})
+</script>
+
+<template>
+  <div :class="$style.shopInfo">
+    <div :class="$style.review">
+      <div>店名: {{ shopDetail.name }}</div>
+      <div>説明: {{ shopDetail.description }}</div>
```

Comment:
ここ API を叩いたあとの変数の `eateryDetail` を使うようにしておいてほしいです…！
```suggestion
      <div>店名: {{ eateryDetail.name }}</div>
      <div>説明: {{ eateryDetail.description }}</div>
```